Sources/Target/Windows/Process.cpp

```diff
@@ -259,7 +259,21 @@ ErrorCode Process::wait() {
       continue;
     }
 
-    case EXCEPTION_DEBUG_EVENT:
+    case EXCEPTION_DEBUG_EVENT: {
+      // First-chance exceptions may yet be caught by the debugee, so don't break for them.
+      auto code = de.u.Exception.ExceptionRecord.ExceptionCode;
+      if (de.u.Exception.dwFirstChance &&
+          (code == DS2_EXCEPTION_UNCAUGHT_COM || code == DS2_EXCEPTION_UNCAUGHT_USER || code == DS2_EXCEPTION_UNCAUGHT_WINRT)) {
```

Comment:
Should we be filtering the code or should we be ignoring all first-chance exceptions?

Reply:
Doing the latter doesn't work, b/c e.g. apparently some first-chance exceptions (looking at you, DS2_EXCEPTION_VC_THREAD_NAME_SET) need to be continued with DBG_CONTINUE. 
I started down that path before deciding that the rest of the roster of codes is different and so this is the least unpalatable option IMO.

Reply:
Updated per tuple.